src/main/scala/ai/privado/threatEngine/PIIShouldNotBePresentInMultipleTablesWithSQL.scala

```diff
@@ -0,0 +1,88 @@
+package ai.privado.threatEngine
+
+import ai.privado.cache.TaggerCache
+import ai.privado.model.{Constants, PolicyOrThreat}
+import ai.privado.model.exporter.ViolationProcessingModel
+import ai.privado.semantic.Language.*
+import ai.privado.threatEngine.ThreatUtility.{getPIINameFromSourceId, hasDataElements}
+import io.shiftleft.codepropertygraph.generated.Cpg
+import io.shiftleft.codepropertygraph.generated.nodes.{SqlColumnNode, StoredNode}
+import io.shiftleft.semanticcpg.language.*
+import org.slf4j.LoggerFactory
+
+import java.util.UUID
+import scala.collection.mutable
+import scala.collection.mutable.ListBuffer
+import scala.util.Try
+
+object PIIShouldNotBePresentInMultipleTablesWithSQL {
+
+  private val logger = LoggerFactory.getLogger(getClass)
+
+  /** Check for violation for data leakage to logs threat - consumes already generated dataflows
+    *
+    * @param cpg
+    *   cpg
+    * @return
+    */
+  def getViolations(
+    threat: PolicyOrThreat,
+    cpg: Cpg,
+    taggerCache: TaggerCache
+  ): Try[(Boolean, List[ViolationProcessingModel])] = Try {
+    val violatingFlows = ListBuffer[ViolationProcessingModel]()
+
+    if (hasDataElements(cpg)) {
+      val taggedSources = getSources(cpg)
+      taggedSources.foreach(groupedSource => {
+        val tableNames           = groupedSource._2.collect(col => col.sqlTable.get.name)
```

Comment:
As SQLTable here is option, better to not use `get` and have proper handling

Reply:
Handled better with `getOrElse`: https://github.com/Privado-Inc/privado-core/pull/814/commits/1f896170d831e6bfac21ffbb33348b16be85211c#diff-e52d364abd76eb28a88e94a14440d5bc178f382802edceeec8130ad885fb9d6aR38